README.md

```diff
@@ -1 +1,182 @@
-# finalGroupCapstone
\ No newline at end of file
+# 🏩 finalGroupCapstone-backend 
```

Comment:
- [ ] This PR has had no changes since the last review.   This is a summary of the issues:

### 1. When running the tests according to instructions, they don't work:

![image](https://github.com/IvonneBenitesRodriguez/finalGroupCapstone-backend/assets/57421823/7850165d-3f6b-4051-a40c-148b3d2c3e68)

### 2. The swagger documentation does not work:

![image](https://github.com/IvonneBenitesRodriguez/finalGroupCapstone-backend/assets/57421823/f0388d75-2f9c-408d-9e18-8698bc271fba)

![image](https://github.com/IvonneBenitesRodriguez/finalGroupCapstone-backend/assets/57421823/a9c1c13c-a0a2-43a0-aca5-2e7cb46bc6d4)